frontend/src/components/documentation/Routes/Workspaces/GetSurveys.jsx

```diff
@@ -121,15 +144,35 @@ export default class WorkspacesGetSurveys extends React.Component {
                             example="false"
                             description="Whether the survey represents a staff workspace (`true`) or a student work or study space (`false`). This is useful in apps where you set the `survey_filter` parameter to `all` and wish to do filtering of student and staff workspaces in your app or API as opposed to leaving the filtering to UCL API. By default, as `survey_filter` is set to `student` unless you specify otherwise, all workspaces will have this parameter set to `false`." />
                         <Cell
-                            name="name"
+                            name="location"
+                            extra="dictionary"
+                            example='{"coordinates": {"lat": "51.522897","lng": "-0.127864"},"address": ["Newsam Library and Archives","20 Bedford Way","London","WC1H  0AL"]'
+                            description="Dictionary of survey location information including co-ordinates and the address." />
+                        <Cell
+                            name='location["coordinates"]'

```

Comment:
This (and not only this) name is also inconsistent with the rest of the docs. 
This endpoint should be good enough for you to understand how endpoint responses are usually represented:
https://uclapi.com/docs#timetable/bymodule

Reply:
The whole docs are inconsistent, for example in that link we do not show the attributes of the Lecturer Object, but we do for other Objects? If you mean the quotes they will be removed i didn't notice we don't include those (even though it makes more sense to me).